roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -171,19 +171,19 @@ backing subsystem itself.
 - `StatementDistributionMessage::NetworkBridgeUpdate`
   - Handled by `handle_network_update`
   - v1 compatibility
-  - `Statement`
-    - Notification of a signed statement.
-    - Handled by `handle_incoming_statement`
-  - `BackedCandidateManifest`
-    - Notification of a backed candidate being known by the sending node.
-    - For the candidate being requested by the receiving node if needed.
-    - Announcement
-    - Handled by `handle_incoming_manifest`
-  - `BackedCandidateKnown`
-    - Notification of a backed candidate being known by the sending node.
-    - For informing a receiving node which already has the candidate.
-    - Acknowledgement.
-    - Handled by `handle_incoming_acknowledgement`
+- `Statement`
+  - Notification of a signed statement.
+  - Handled by `handle_incoming_statement`
+- `BackedCandidateManifest`
+  - Notification of a backed candidate being known by the sending node.
+  - For the candidate being requested by the receiving node if needed.
+  - Announcement
+  - Handled by `handle_incoming_manifest`
+- `BackedCandidateKnown`
+  - Notification of a backed candidate being known by the sending node.
+  - For informing a receiving node which already has the candidate.
+  - Acknowledgement.
+  - Handled by `handle_incoming_acknowledgement`
```

Comment:
These are the possible messages for `NetworkBridgeUpdate`, which is why they were indented like that. I see it's a bit confusing. Maybe we can keep your indentation and just add a new section, like this:

```md
- `StatementDistributionMessage::NetworkBridgeUpdate`
  - Handled by `handle_network_update`
  - Possible events: see next section.

#### Network bridge events

- v1 compatibility
- `Statement`
  - Notification of a signed statement.
[...]
```